R/pkg/R/mllib_classification.R

```diff
@@ -239,21 +253,57 @@ function(object, path, overwrite = FALSE) {
 setMethod("spark.logit", signature(data = "SparkDataFrame", formula = "formula"),
           function(data, formula, regParam = 0.0, elasticNetParam = 0.0, maxIter = 100,
                    tol = 1E-6, family = "auto", standardization = TRUE,
-                   thresholds = 0.5, weightCol = NULL, aggregationDepth = 2) {
+                   thresholds = 0.5, weightCol = NULL, aggregationDepth = 2,
+                   lowerBoundsOnCoefficients = NULL, upperBoundsOnCoefficients = NULL,
+                   lowerBoundsOnIntercepts = NULL, upperBoundsOnIntercepts = NULL) {
             formula <- paste(deparse(formula), collapse = "")
+            lrow <- 0
+            lcol <- 0
+            urow <- 0
+            ucol <- 0
```

Comment:
Question: Based on my understanding, `lowerBoundsOnCoefficients ` and `upperBoundsOnCoefficients ` are not required to set at the same time. They can be set at the same time.
For the first case, we can't enforce the dimension of the two matrices because one could be `NULL`. 
For the second case, we can check it.

So, we can't enforce the rule in general.